File: pytools/examples/php/server/protocol/AckRoleLoginOkNoRole.php
<?php
namespace protocol;

use network\Packet;


class AckRoleLoginOkNoRole
{


	public function encode()
	{
		$packet = new Packet();
		return $packet->encode(Msg::$P_ACK_ROLE_LOGIN_OK_NO_ROLE);
	}



}

File: pytools/examples/java/server/protocol/AckRoleLoginOkNoRole.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class AckRoleLoginOkNoRole
{


	public byte[] encode()
	{
		Packet packet = new Packet();
		return packet.encode((short)Msg.P_ACK_ROLE_LOGIN_OK_NO_ROLE);
	}



}
